package.json

```diff
@@ -10,10 +10,11 @@
     "build": "webpack",
     "lint": "eslint -- .",
     "lint:fix": "eslint --fix -- .",
```

Comment:
Why `lint` and `lint:fix` use `eslint --fix` but `lint-staged` uses `prettier --write`?